docs/software/comanage-troubleshooting-guide.md

```diff
@@ -0,0 +1,77 @@
+COmanage Troubleshooting Guide
+====
+Introduction
+----
+The purpose of this guide is to be a resource for COmanage administrators to find solutions to commonly encountered problems.
+
+
+COManage Troubleshooting Items
+----
+This section contains some of the issues you may encounter when interacting with the OSG COmanage.
+
+Problem list:
+
+ - [Tiger-Dex Unable to match User to COmanage CO Person identity](#tiger-dex-unable-to-match-user-to-comanage-co-person-identity)
+
+###Tiger-Dex Unable to match User to COmanage CO Person identity
```

Comment:
```suggestion
### Tiger-Dex Unable to match User to COmanage CO Person identity
```

Whitespace for readability. Also I'm not sure if mkdocs parses the heading without it

Also, this is more of the root cause of the problem rather than the problem itself. The problem is along the lines of "User is not authorized to access Tiger with Dex credentials"